docs/docs/configuration.md

```diff
@@ -67,7 +67,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.metadata.metrics.column.col1                   | (not set)                   | Metrics mode for column 'col1' to allow per-column tuning; none, counts, truncate(length), or full                                                                                                |
 | write.target-file-size-bytes                         | 536870912 (512 MB)          | Controls the size of files generated to target about this many bytes                                                                                                                              |
 | write.delete.target-file-size-bytes                  | 67108864 (64 MB)            | Controls the size of delete files generated to target about this many bytes                                                                                                                       |
-| write.distribution-mode                              | none                        | Defines distribution of write data: __none__: don't shuffle rows; __hash__: hash distribute by partition key ; __range__: range distribute by partition key or sort key if table has an SortOrder |
+| write.distribution-mode | none (see engines for specific defaults) | Defines distribution of write data: __none__: don't shuffle rows; __hash__: hash distribute by partition key ; __range__: range distribute by partition key or sort key if table has an SortOrder |
```

Comment:
I am wondering if it's helpful to hyperlink the engine specific defaults here or in notes , for the convenience of lookup

like
```markdown
[Spark](spark-configuration.md#write-options)
```

Reply:
Added